backend/src/lib/transformer/index.js

```diff
@@ -35,6 +35,7 @@ export default function (key, data, mopidy) {
       return data.new_state
     case Mopidy.GET_CURRENT_TRACK:
       const trackInfo = TransformTrack(data)
+      if (!data) return null
```

Comment:
Answer may well be no - but is there any merit to logging anything here to see how often it happens?

Reply:
This change is only for the very first `getCurrentTrack` call the client makes. It was bailing in the instance that was an empty tracklist i.e no current track. I think the PR in general mitigates any other circumstance where this could happen. I'm sure I may regret saying that.